packages/slider/src/Slider.ts

```diff
@@ -59,32 +58,52 @@ export class Slider extends Focusable {
         return this._variant;
     }
 
+    public get values(): HandleValueDictionary {
+        return this.handleController.values;
+    }
+
+    public get handleName(): string {
+        return 'value';
+    }
+
     /* Ensure that a '' value for `variant` removes the attribute instead of a blank value */
     private _variant = '';
 
-    @property({ attribute: false })
-    public getAriaValueText: (value: number) => string = (value) => `${value}`;
+    @property({ type: String, reflect: true })
+    public language: string = navigator.language;
```

Comment:
As I mentioned in #1495, my application doesn't necessarily use the value from `navigator.language`. So, I added this here. It seems a little painful to have to provide this to each component though. I wonder if there's a better API we could provide to allow override of `navigator.language` for all SWC components on the page?

Reply:
Does this specifically need to reflect?

Reply:
Nope.